packages/opentelemetry-sdk-trace-base/test/common/export/BatchSpanProcessorBase.test.ts

```diff
@@ -32,12 +32,12 @@ import {
   Span,
   SpanExporter,
 } from '../../../src';
-import { context } from '@opentelemetry/api';
+import { Attributes, context } from '@opentelemetry/api';
```

Comment:
note for reviewer: leftover from https://github.com/open-telemetry/opentelemetry-js/pull/5009